stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.24.1
+version: 1.25.0
```

Comment:
Something else got in ahead of this. This Is going to have to b1 `1.26.0`. After that, LGTM and we'll get it merged.

Reply:
Alright, will change and rebase!